libs/application/templates/directorate-of-immigration/citizenship/src/fields/SelectedChildrenExtra/SelectedRepeaterItem.tsx

```diff
@@ -140,7 +140,7 @@ export const SelectedRepeaterItem: FC<Props & FieldBaseProps> = ({
           setShowMoreQuestions(!showMoreQuestions)
         }}
         disabled={readOnlyFields}
-        defaultValue={repeaterField.hasFullCustody === 'yes' ? YES : ''}
+        defaultValue={repeaterField.hasFullCustody === NO ? NO : ''}
```

Comment:
Could it be that this is suppose to be YES, or was it wrong before

Reply:
yeah it was wrong before